CHANGELOG.md

```diff
@@ -24,6 +24,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - `Binarizer` preprocessing model
 - Preprocessing models test runner
 ### Changed
+- `Command.DESERIALZIE` renamed to `Command.DESERIALIZE`
+- `SerilaizatoinErrorTypes` renamed to `SerializationErrorTypes`
+- `DeSerilaizatoinErrorTypes` renamed to `DeserializationErrorTypes`
```

Comment:
@sepandhaghighi @sadrasabouri 
I think the format in which we write in `CHANGELOG.md` is not compatible with these lines.
I mean, isn't it better to say that we have an update in `transporter.py` that we have the base definition of the `Command` enum class?

maybe something like this:
- `Command` enum class in `transporter.py`